Source/cmake/OptionsGTK.cmake

```diff
@@ -357,9 +359,20 @@ if (USE_GBM)
 endif ()
 
 if (ENABLE_SPEECH_SYNTHESIS)
-    find_package(Flite 2.2)
-    if (NOT Flite_FOUND)
-        message(FATAL_ERROR "Flite is needed for ENABLE_SPEECH_SYNTHESIS")
+    if (USE_SPIEL)
+        find_package(LibSpiel)
+        if (NOT LibSpiel_FOUND)
+            message(FATAL_ERROR "LibSpiel is needed for ENABLE_SPEECH_SYNTHESIS")
+        endif ()
+        SET_AND_EXPOSE_TO_BUILD(USE_SPIEL ON)
```

Comment:
is this needed? USE_SPIEL is already ON here.

Reply:
Yes it's needed. Other USE_* cases do the same. This adds the USE_ define in cmakeconfig.h which (afaik) is then used to compute USE(...) macros.